plugins/table/lang/af.js

```diff
@@ -26,7 +26,9 @@ CKEDITOR.plugins.setLang( 'table', 'af', {
 		bgColor: 'Agtergrondkleur',
 		borderColor: 'Randkleur',
 		data: 'Inhoud',
-		header: 'Opskrif',
+		header: 'Header', // MISSING
```

Comment:
Currently, we lost translations for the `header`. Probably we would need to restore versions of lang files from before your changes and then generate entries for the two new items.

Reply:
Right. I reverted two commits with lang files & generated new ones.